_targets.R

```diff
@@ -3,66 +3,62 @@ source("1_fetch/src/get_nwis_data.R")
 source("2_process/src/process_and_style.R")
 source("3_visualize/src/plot_timeseries.R")
 
+# Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
 options(tidyverse.quiet = TRUE)
-tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
+tar_option_set(packages = c("tidyverse", "dataRetrieval", "arrow"))
 
 p_width <- 12
 p_height <- 7
 p_units <- "in"
-parameterCd = '00010'
-startDate="2014-05-01"
-endDate="2015-05-01"
+parameterCd <- '00010'
+startDate <- "2014-05-01"
+endDate <- "2015-05-01"
 
 p1_targets_list <- list(
   tar_target(
-    site_data_01427207_csv,
-    download_nwis_site_data(filepath =  "1_fetch/out/nwis_01427207_data.csv",
+    site_data_01427207,
```

Comment:
Nice work catching this change to the target name, now that it's no longer a file target